### ISSUE TITLE
IMAP email fetching is not detecting new messages since latest Chatwoot update

### ISSUE THREAD

**Issue by silbox-ch** [chatwoot/chatwoot#8829]
### Describe the bug

Email fetching task is properly connecting to the IMAP inbox, it detects some old messages, but not the new ones.

### To Reproduce

Cloudron app version: 1.24.2
Chatwoot upstreamVersion: 3.5.2

### Expected behavior

New incoming emails should be detected

### Environment

Other [please specify in the description]

### Cloud Provider

None

### Platform

Browser

### Operating system

Cloudron 7.6.3

### Browser and version

Firefox latest

**Comment by frerej** [comment]
Here's an extract from the log.

Feb 01 12:52:03I, [2024-02-01T11:52:03.766953 #15] INFO -- : [ActiveJob] [Inboxes::FetchImapEmailsJob] [b8b36699-9d99-4559-bf21-1fadd7adc5c1] [IMAP::FETCH_EMAIL_SERVICE] Fetching mails from @.ch, found 5.

It found 5 emails. If I send an email to this mailbox, it doesn't found 6 emails. Stuck at 5.

**Comment by sojan-official** [comment]
we gave a possible fix for this in : https://github.com/chatwoot/chatwoot/pull/8833/files

**Comment by frerej** [comment]
Ok, will this fix be included in the next release?

**Comment by vishnu-narayanan** [comment]
@frerej Yes, it is already merged and will be available in v3.6 release.

**Comment by jordan26** [comment]
Hey @frerej , I too need to check my IMAP logs etc. How/where do you find the logs in general for chatwoot? I am SSH'd in, what command(s) need to be run to see logs?

**Comment by frerej** [comment]
Hi @jordan26 
I use Cloudron, so the logs are accessible through the app settings.

**Comment by vishnu-narayanan** [comment]
@jordan26 Logs depend on your installation setup. If you are on a linux setup, use `cwctl`. 

```
cwctl --logs web
cwctl --logs worker
```

https://www.chatwoot.com/docs/self-hosted/deployment/chatwoot-ctl/#viewing-logs

**Comment by rohanchauhan02** [comment]
I am also face the similar issue, 
![Uploading Screenshot 2024-02-07 at 2.40.20 PM.png…]()

**Comment by rohanchauhan02** [comment]
Is there any chance that the issue is happening due to redis connection timeout ??

**Comment by rohanchauhan02** [comment]
I am still facing this issue on v3.6.

**Comment by frerej** [comment]
@rohanchauhan02 me too. I’ve abandoned Chatwoot and migrated to Crisp. Much better, for a reasonable cost.

**Comment by jarne** [comment]
Same issue here. Some e-mails are "stuck" in the processing queue and show over and over again in these logs. I can also see them in the actual IMAP inbox, but they don't show up in Chatwoot. There is no error message in the logs, they just appear there but are not imported.

<img width="1300" alt="Bildschirmfoto 2024-03-04 um 16 24 57" src="https://github.com/chatwoot/chatwoot/assets/15860314/6bd39181-9310-4e8b-83bc-9dae62922599">

Super annoying, as it's not acceptable to just "loose" some customer e-mails randomly.

**Comment by sojan-official** [comment]
Closing this as stale since the original report was for v3.5.2 and there have not been recent reproducible reports with current Chatwoot versions.

A related fix was merged in [#8833](https://github.com/chatwoot/chatwoot/pull/8833), but the later comments suggest there may have been more than one cause involved.

Please open a new issue if this still reproduces on a current Chatwoot version, including:

- Chatwoot version and deployment method
- IMAP provider
- Worker/Sidekiq logs from `Inboxes::FetchImapEmailsJob`
- Whether Redis connection/read timeout errors are present around the same time
- Whether the stuck emails have large attachments, unusual encodings, or forwarded `.eml` messages
